fuse/fuse.h

```diff
@@ -20,12 +20,39 @@
 #define FUSE_H
 
 #include <string>
+#include <vector>
 
-int run_in_fuse(
-	const std::string& daemon_path,
-	const std::string& working_dir,
-	const std::string& json,
-	bool use_stdin_file,
-	std::string& result_json);
+#ifdef __linux__
+#include "namespace.h"
+#endif
+
+struct json_args {
+	std::vector<std::string> command;
+	std::vector<std::string> environment;
+	std::vector<std::string> visible;
+	std::string directory;
+	std::string stdin_file;
+
+#ifdef __linux__
```

Comment:
Please make these unconditionally parsed.
We can still choose to ignore them later.